app/models/trip.rb

```diff
@@ -85,4 +85,45 @@ def self.date_with_highest_trips
   def self.date_with_lowest_trips
     date = group(:start_date).order("count_id ASC").count(:id).first
   end
+
+  def self.rides_started(id)
```

Comment:
This should probably live on Station instead of trip. See my comment on the view for more details, but moving this method should make it so that you don't have to pass `id` as an argument.